src/Fetcher/ConferencesHarvester.php

```diff
@@ -0,0 +1,222 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Fetcher;
```

Comment:
Not the right namespace IMHO.
The harvester **use** a fetcher, but it **isn't** a Fetcher.
I would use `Conference` NS instead